### ISSUE TITLE
Buff C4 dmg to buildings and bigger range dmg to players

### ISSUE THREAD

**Issue by Haox09** [deferred-impact/battledudes-issue-tracker#119]
The c4 damage range to buildings is just to small. The exposion is comparable to a sticky grenade. I think that C4 should have a larger range of damage to buildings while enlarge the radius that will deal damage to players. Damage to players should be like this: point blank--> right next to it is fatal while anything from 1 block to 10 blocks (maybe smaller) is survivable, the damage smaller the further to stand. Damage to buildings: all wall near the c4 explosion (5blocks) should be destroyed. To make the weapon not to be too op, the throw range now should be kept. C4's use is just too low and it have so much potential to be a hardpoint-wallbreaker.

**Comment by Gregory-Y** [comment]
It can already be a good hardpoint wallbreaker. Just distribute a bunch of them near a section of the wall and boom. I do agree they should deal more damage so they can destroy the thick metal walls easily but no need to buff the area otherwise the entire map would have no cover.

**Comment by deferred-impact** [comment]
I like this idea partially.

Yes, i would like some throwables to be better against walls than others. (To some extent we have it: the Rocket is able to precisely remove walls, but is not viable for large scale removal. Between grenades, however, there is little diversity in this regard). Whether C4 should be the throwable that gets the bonus, is certainly up to debate.

But i do not want walls to be even easier to destroy, come on, dude! You played Compound recently? Sure, not having to deal with roofs - love it. But all walls get erased within 5 minutes! That is not desirable in the slightest.

Basically, all i want, is instead of buffing C4 wall damage, we nerf the wall damage of other throwables.

**Comment by Haox09** [comment]
finally a aproved defered idea
Gregory the problem is that it have a small range of player dmg which is not accurate. 
To defered's problem maybe the walls should have more hp or gun deal less dmg to walls but c4 should be a better source of wallbreaking. The situation rn is like people tend to not use throwables to break walls but to use it to kill other players. Thus the buff to be more usefull.

**Comment by deferred-impact** [comment]
There are plenty of ideas i approve of :moyai: most famously (and stupidly) the crab launcher #88 

>  Gregory the problem is that it have a small range of player dmg which is not accurate.

Well, that is definitely to be debated. Fragmentation flies further than the shockwave, so it may be more realistic to have frag grenades have larger player damage AOE (the C4 produces virtually no fragmentation)

**Comment by Haox09** [comment]
virtually no but irl c4 have a much bigger range. i know you hate implementing real life characteristics into game but the c4 one should be bigger than grenade
on that note buff grenade dmg also like wider range but the further the less dmg
maybe make c4 have a bigger range of small smg area

**Comment by deferred-impact** [comment]
> c4 have a much bigger range

I call complete BS on that. Frag grenades have fragmentation areas in hundreds of meters. Sure, it's luck whether any fragment will actually choose to go towards you, but it's at least an order of magnitude higher than what half a kilo of C4 would offer. Correct?

**Comment by Haox09** [comment]
i mean in my remembering c4 have a larger range to buildings but not nessesary to players? Like i actually don't know much about how c4 works irl
